utilities/AFQ_mrtrixInit.m

```diff
@@ -1,4 +1,4 @@
-function files = AFQ_mrtrixInit(dt6, lmax, mrtrix_folder)
+function files = AFQ_mrtrixInit(dt6, lmax, mrtrix_folder, mrtrixVersion)
```

Comment:
OK, I will check it in the server instead of my local osx machine. Are you going to make changes to the other files or should I commit everything back?